package.json

```diff
@@ -14,6 +14,7 @@
     "ajv": "8.17.1",
     "ajv-formats": "3.0.1",
     "axios": "1.7.7",
+    "better-npm-audit": "3.11.0",
```

Comment:
Although I can see the advantages of this package, I have a few concerns:

- It is itself an npm dependency that depends on a number of packages that may have vulnerabilities
- According to the [Readme](https://www.npmjs.com/package/better-npm-audit?activeTab=readme) , they are looking for collaborators, which suggests that it may not be well maintained (although I do see a recently published version)
- `npm` seems to change the audit command behavior between major versions, which also requires major changes to this package

Instead could we:

1. not depend on `npm audit` in Pull Requests?
2. run `npm audit` in a non-blocking way?
3. run `npm audit` on a schedule?